library/spdm_requester_lib/libspdm_req_vendor_request.c

```diff
@@ -65,6 +66,21 @@ libspdm_return_t libspdm_try_vendor_send_request_receive_response(
 
     transport_header_size = spdm_context->local_context.capability.transport_header_size;
 
+    if ((req_size > SPDM_MAX_VENDOR_DEFINED_DATA_LEN) &&
```

Comment:
Need to return an error if `req_size` is larger than `65535` and Requester and/or Responder has not set `LARGE_RESP_CAP`.